rotkehlchen/serialization/deserialize.py

```diff
@@ -535,6 +558,7 @@ def deserialize_evm_transaction(
         to_address = deserialize_evm_address(data['to']) if is_empty_to_address else None
         value = read_integer(data, 'value', source)
 
+        authorization_list: list[EvmTransactionAuthorization] | None = None
```

Comment:
this shouldn't be required. Can you figure out why? It is a problem because you do an assign that is not required. Do you have dependencies up to date?